app/main/views/suppliers.py

```diff
@@ -77,7 +77,24 @@ def dashboard():
     ), 200
 
 
-@main.route('/edit', methods=['GET'])
```

Comment:
~~~Oh forgot about that - I'll do it anyway, well caught.~~~

Actually come to think of it I think we only ever wanted to put the 302 in to make the temporary halfway situation work. And we decided to do away with the temporary situation.